src/views/calculator.tsx

```diff
@@ -11,15 +11,27 @@ const Calculator = () => {
     result: 0,
   });
 
+  const [pastEquations, setPastEquations] = useState([<></>]);
+
   return (
     <div>
-      <header className="header">
-        <p>Options</p>
-      </header>
-      <Display mainString={calc.displayedString} result={calc.result} />
+      <Display
+        mainString={calc.displayedString}
+        result={calc.result}
+        pastEquations={pastEquations}
+      />
       <div className="PadBtnGrid">
```

Comment:
[fix] Whenever you're rendering a list of items you should use either `ul`, for a list that has no particular order, or `ol`, for lists where the elements have some order or hierarchy,  and `li` for the children elements. This makes your code more semantically correct.